pyproject.toml

```diff
@@ -32,6 +32,12 @@ skip_glob = [
     'kubernetes/*'
 ]
 
+[tool.mypy]
```

Comment:
I've been running this locally... I think it'll have limited utility for the time being because neither luigi nor hail has the right support.